### DISCUSSION TITLE
Proxies

### DISCUSSION THREAD

**Discussion by Terrinn** [3kh0/ext-remover#249] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Any proxies I can use that are 100% not blocked and not a repl aka fast?

**Comment by ILikeBypassing** [comment, marked as answer]
https://amongusayo.herokuapp.com/1.html
my holy unblocker site, you can deploy your own btw

**Reply by Terrinn** [reply]
Heroku is blocked for me, anyone know anything else?

**Reply by ILikeBypassing** [reply]
just deploy your own from the holy unblocker repo on git

**Reply by Terrinn** [reply]
Can you send me a link for that?

**Reply by Terrinn** [reply]
I found it but it doesn't work on repl and everything else is blocked

**Reply by ILikeBypassing** [reply]
thats a rip

**Reply by md4r2** [reply]
you could use other services like render, cyclic etc

**Reply by rwg100** [reply]
"among us ayo"
man why you do the url name like that bruh

**Comment by Ironbrodiet** [comment]
Heroku is blocked by my school district, can you give me some other links?

**Reply by hoby-lang** [reply]
heres one 
https://itoldyoulongago.onrender.com/
